examples/next/docs/react/basic-example/config-overrides.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (config, ...rest) => {
+  return { ...config, resolve: { ...config.resolve, symlinks: false } };
```

Comment:
https://github.com/handsontable/handsontable/pull/7588#discussion_r586335601 @Osky772 maybe here we can use Webpack option to change the output dir